src/MarkdownTextInput.web.tsx

```diff
@@ -741,6 +741,7 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
         onCopy={handleCopy}
         onCut={handleCut}
         onPaste={handlePaste}
+        // @ts-expect-error: we use placeholder attr to style it in css even though its not handled internally
```

Comment:
I'm not sure what this comment means, could you please explain it?
```suggestion
        // @ts-expect-error: we use placeholder prop to style it in CSS even though its not handled internally
```